### DISCUSSION TITLE
Welcome to Hackerrank-Python Discussions!

### DISCUSSION THREAD

**Discussion by adityajai25** [adityajai25/Hackerrank-Python#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by aekostadinov** [comment]
Hello, I'm learning Python from the summer of 2022.Now I'm interested in JS, HTML/CSS also.Currently I'm interested with DJANGO Web Framework and searching people to collaborate with.

**Reply by adityajai25** [reply]
Yeah right place for collab
